modules/base/markdown.go

```diff
@@ -346,7 +340,7 @@ OUTER_LOOP:
 func RenderMarkdown(rawBytes []byte, urlPrefix string, metas map[string]string) []byte {
 	result := RenderRawMarkdown(rawBytes, urlPrefix)
 	result = PostProcessMarkdown(result, urlPrefix, metas)
-	result = Sanitizer.SanitizeBytes(result)
+	result = BuildSanitizer().SanitizeBytes(result)
```

Comment:
Yes. And like I mentioned in #2396, there's no way to build it on start up, since the config isn't loaded then.
I'll try doing it another way and see if it works